clang/lib/Analysis/LifetimeSafety/Facts.cpp

```diff
@@ -95,6 +95,30 @@ void FactManager::dump(const CFG &Cfg, AnalysisDeclContext &AC) const {
   }
 }
 
+void FactManager::dumpBlockSizes(const CFG &Cfg,
```

Comment:
I do not think this dump method is super useful after this PR lands. I would drop this for now in this PR.
In a separate PR, as part of LifetimeStats, we could have `NumFunctionsAnalysed`, `NumFunctionsSkipped` and can be printed as part of -print-stats.